### ISSUE TITLE
Please write down your ESC TODO and what you did on this sheet 🙇

### ISSUE THREAD

**Issue by cardanoz** [C3ETH/c3eth-operations#341]
Please write down your ESC TODO and what you did on this sheet 🙇

@C3ETH/c3eth-team

**Comment by Linh-LP** [comment]
Workshops: - VN - Provided info for last 6 sessions and planned for next 12 sessions--> please click on Excel file and and see comments accommodated with colour coded cells.
https://docs.google.com/spreadsheets/d/189K_BOlNIJ3s5AhNjZoJaT-cP5anDwbo/edit#gid=1402193534 (VN need to do 6 more workshops not 12)

**Comment by cardanoz** [comment]
thank you so much ! 
and I've consolidated them into a KPI spread sheet so I will close this one 🙇

**Comment by andreassosilo** [comment]
I did update the Google Sheet based on the activities from Team Indonesia. We did 6x workshops for Fund 9 and gave 10x 1-on-1 session in Fund 9.

We will deliver another 6x workshops in Fund 10 and more 1-on-1 session.

Anyway our KPI is to deliver in total 36x workshops, divided into three team. So Vietnam needs to do 12x workshops, Indonesia 12x workshops and Japan 12x workshops in Fund 9 & Fund 10.
